plugins/out_es/es.c

```diff
@@ -189,10 +190,15 @@ static char *elasticsearch_format(void *data, size_t bytes,
 
     /* If logstash format and id generation is disabled, pre-generate index line for all records. */
     if (ctx->logstash_format == FLB_FALSE && ctx->generate_id == FLB_FALSE) {
+        flb_time_get(&tms);
+        gmtime_r(&tms.tm.tv_sec, &tm);
+        s = strftime(index_formatted, sizeof(index_formatted) - 1,
+                     ctx->index, &tm);
+        es_index = index_formatted;
         index_len = snprintf(j_index,
                              ES_BULK_HEADER,
                              ES_BULK_INDEX_FMT,
-                             ctx->index, ctx->type);
+                             es_index, ctx->type);
```

Comment:
is this ok when ctx->current_time_index == FLB_FALSE ?

Reply:
@onorua Did you address this question?

Reply:
Nevermind. Looks not relevant in the latest.